### PR TITLE
chore: Migrate Slack source to MD, fix integrations build

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Part of https://github.com/cloudquery/cloudquery-issues/issues/620 (internal issue)

When we render integrations those are always rendered as MDX, so some of the MD syntax like `{` broke them, so I had to escape it.

### CODE REVIEW

File: website/next.config.mjs [hunk 1]
```diff
@@ -16,7 +16,7 @@ const pluginNamePatterns = {
 };
 
 const getKindAndName = (file) => {
-  const match = file.history[0].match(/pages\/docs\/plugins\/(.+)\/(.+)\//);
+  const match = file.history[0].match(/\/plugins\/(.+)\/(.+)\//);
```
Comment (erezrokah) [inline website/next.config.mjs:19]: So we can use directives when rendering integrations

File: website/next.config.mjs [hunk 2]
```diff
@@ -92,25 +92,23 @@ const customPlugin = () => {
         const hast = h(node.name, node.attributes || {});
         data.hName = hast.tagName;
         data.hProperties = hast.properties;
-        if (!['badge', 'configuration', 'authentication', 'callout'].includes(data.hName)) {
+        if (!['badge', 'configuration', 'authentication', 'callout', 'slack-app-link'].includes(data.hName)) {
           return;
         }
 
         const { kind, name } = getKindAndName(file);
```
Comment (erezrokah) [inline website/next.config.mjs:99]: Unrelated cleanup

File: website/next.config.mjs [hunk 3]
```diff
@@ -92,25 +92,23 @@ const customPlugin = () => {
         const hast = h(node.name, node.attributes || {});
         data.hName = hast.tagName;
         data.hProperties = hast.properties;
-        if (!['badge', 'configuration', 'authentication', 'callout'].includes(data.hName)) {
+        if (!['badge', 'configuration', 'authentication', 'callout', 'slack-app-link'].includes(data.hName)) {
```
Comment (erezrokah) [inline website/next.config.mjs:95]: We used MDX to render a nice Slack Install App link

**Review by disq — APPROVED** [review]

File: website/scripts/prebuild.tsx [hunk 4]
```diff
@@ -113,20 +113,22 @@ function recreateDirectory(dir: string) {
 function copySourceAuthenticationFile(source: Plugin) : boolean {
     const sourceDir = `./pages/docs/plugins/sources/${source.id}`;
     // Copy the authentication and configuration files if they exist
-    const authFilePath = locatePathSync([`${sourceDir}/_authentication.mdx`, `${sourceDir}/_authentication.md`]);
+    const authFilePath = locatePathSync([`${sourceDir}/_authentication.md`]);
```
Comment (erezrokah) [inline website/scripts/prebuild.tsx:116]: Cleanup MDX related files

File: website/utils/plugins.js [hunk 5]
```diff
@@ -4,24 +4,9 @@ const matter = require("gray-matter");
 const title = require("title");
 const pluginTitle = (plugin) => title(plugin.replace(/-/g, " "));
 
-const readPluginOverview = (pluginsDir, plugin) => {
```
Comment (erezrokah) [inline website/utils/plugins.js:7]: Clean up MDX related stuff
